_docs/contribute/dev-process.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page-dev
+title: Overview of the FIO Development Process
+description: Overview of the Process for Contributing to FIO
+---
+
+# Overview of the FIO Community's Development Process
```

Comment:
Remove this section. FIO no longer operates as a horizontal "development group". Instead, each project (dashboard, registration, core chain), operates as an independent worker proposal led by a worker proposal lead. So, publishing a generic development process does not make sense.